source/common/common/logger.h

```diff
@@ -100,12 +100,15 @@ class Registry {
   static void initialize(uint64_t log_level, Thread::BasicLockable& lock);
 
   /**
-   * @return const std::vector<Logger>& the installed loggers.
+   * @return const std::vector<Logger>& return a const reference to the installed loggers.
```

Comment:
Nit: I think the change is a bit redundant, since this information was there before.